### DISCUSSION TITLE
dosbox-x on macos sonoma?

### DISCUSSION THREAD

**Discussion by dlrubin** [joncampbell123/dosbox-x#4766] [Q&A]
Is there a build of dosbox-x that runs on sonoma? When I launch the current macos release of dosbox-x, it quits with error saying the application quite unexpectedly.

**Comment by heafnerj** [comment]
I get the same error message and I've tried installing via Homebrew and the official releases. The library libavcodec.60.dylib seems to be missing from various places. I'm trying to run dosbox-x on my Intel MacBook Pro running Sonoma 14.3.1. I opened an issue.

**Comment by heafnerj** [comment]
The following nightly build seems to fix the issue. The issue wasn't fixed until the after the latest official release.

https://github.com/joncampbell123/dosbox-x/actions/runs/7876516635

**Reply by dlrubin** [reply]
Thanks so much, heafnerj!  Though I haven't installed builds before. Any guidance on how to do this?  Thanks!

**Reply by heafnerj** [reply]
The link I posted will take you to the build I downloaded. You install it just like the official releases.

**Reply by dlrubin** [reply]
Ah, I missed the second link. Thanks so much for doing this! Looking forward to seeing this in the next official release.

**Reply by heafnerj** [reply]
I am too!
